src/tests/test_magic_plot.py

```diff
@@ -80,6 +84,20 @@ def test_validate_arguments(tmp_empty, ip, cell, error_type, error_message):
                 reason="problem in IPython.core.magic_arguments.parse_argstring",
             ),
         ),
+        pytest.param(
+            "%sqlplot bar --table spaces.csv --column 'some column'",
+            marks=pytest.mark.xfail(
+                sys.platform == "win32",
+                reason="problem in IPython.core.magic_arguments.parse_argstring",
```

Comment:
On Windows, we can use

`'%sqlplot boxplot --table spaces.csv --column "some column"'`